shuup/admin/modules/orders/json_order_creator.py

```diff
@@ -157,7 +162,7 @@ def _process_lines(self, source, state):
             try:
                 self._add_json_line_to_source(source, sline)
             except Exception as exc:  # pragma: no cover
-                self.add_error(exc)
+                self.add_error(ValidationError(str(exc), code="line_error"))
```

Comment:
Validation error should be (a) unicode (on py2) and (b) translated (user visible) message.